src/core/errors/index.ts

```
@@ -0,0 +1,550 @@
+        statusCode: options.statusCode,
+        context: options.context,
+        cause: options.cause,
+      });
+    case 'configuration':
+      return new ConfigurationError(options.message, {
+        code: options.code,
+        statusCode: options.statusCode,
+        context: options.context,
+        cause: options.cause,
+      });
+    case 'generic':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)